### ISSUE TITLE
An orizontal profil list on top of all settings screen

### ISSUE THREAD

**Issue by Vucumpra** [osmandapp/OsmAnd#18918]
### Describe the idea (required)

Hello, I regularly modify some parameter for testing or by occasional necessity.
When I want to put the same value on another profil, it's annoying to go back on the top of tree and search again the parameter to change.
And once returned to the main menu, it's not possible to directly switch to another profil, you have to exit the main menu and reactivate it.

But, I see on the "screen configuration" page (see the picture) a profil list is displayed on top and it's very easy to switch from one profile to another

![SS_20240116_005305-88](https://github.com/osmandapp/OsmAnd/assets/156325356/c5146cae-aded-43e1-ba1b-96b69c62101b)

### Tell us about the expected behaviour (required)

Will be nice to display always a top list profil on every settings screen.

### Tell us about alternatives you've considered (required)

no idea

**Comment by yuriiurshuliak** [comment]
To better understand your request, could you please clarify where exactly you would like the top panel of profiles to be displayed? 
Currently, the "Configure map" and "Configure screen" already have a profile selection bar at the top, considering their profile-dependent nature.

It's important to note that other settings like "Map markers," "My Places," "Download maps," etc., are common to all profiles. They do not have profile-specific configurations. 


https://github.com/osmandapp/OsmAnd/assets/127092082/1023f0b8-6e73-49c7-bcb2-97a42f90416b

**Comment by Vucumpra** [comment]
Ok, for example this another screens :
![SS_20240117_132643-77](https://github.com/osmandapp/OsmAnd/assets/156325356/577780ed-9eb3-469f-89e4-1f20906e5816)
![SS_20240117_132724-77](https://github.com/osmandapp/OsmAnd/assets/156325356/479f8a7c-ba11-4192-be75-6cf97824dc6b)
![SS_20240117_132851-77](https://github.com/osmandapp/OsmAnd/assets/156325356/993f9441-1e78-4d26-b717-95c4b0ada1b5)

I think that is a good idea to switch profil very quickly because OsmAnd it's a great app but a nightmare to configure.
So, once you have discover a good value for your personal use, it will be very useful to put quickly the good value to another profil.

**Comment by yuriiurshuliak** [comment]
You are right, within the Profiles (Settings) section, each profile should be individually configured comprehensively.

I understand that you are proposing the implementation of a top panel of profiles across Profiles (Settings) screens, to enable seamless profile switching, particularly during parameter modifications.
